domains/jeanclaude.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "jeanclaudegeagea",
+        "email": "jeanclaude.geagea@gmail.com"
+    },
+    "record": {
+        "type": "CNAME",
+        "name": "is-a",
+        "value": "jeanclaude-omega.vercel.app"
+    }
```

Comment:
Invalid way to register a domain. See docs for more information: https://docs.is-a.dev/